cmake/common.cmake

```diff
@@ -796,6 +796,8 @@ function(add_picobench name)
                     ccfcrypto.host
   )
 
+  add_san(${name})
```

Comment:
This prevents false positive TSAN warnings that happens when only some object files are compiled with TSAN.